src/util/logging.cpp

```diff
@@ -6,7 +6,7 @@ namespace faabric::util {
 void initLogging()
 {
     // Docs: https://github.com/gabime/spdlog/wiki/3.-Custom-formatting
-    spdlog::set_pattern("[%H:%M:%S:%e] [%t] [%l] %v");
+    spdlog::set_pattern("%^[%H:%M:%S] [%t] [%L]%$ %v");
```

Comment:
This adds colour based on log level back into our logging and makes the format more compact. Logs were quite hard to read before.